files/grest/rpc/assets/asset_policy_list.sql

```diff
@@ -0,0 +1,32 @@
+CREATE FUNCTION grest.asset_policy_info (_asset_policy text)
```

Comment:
We said that we didn't want bulk on this one as we didn't see the use case for that and that it could potentially be very spammy. If we get a request to add bulk we can revisit.